src/app/faq/faq.component.scss

```diff
@@ -0,0 +1,7 @@
+.home-button{
+    font-size: 1.5rem;
+    
+    .material-icons{
+        vertical-align: middle !important;
+    }
+}
```

Comment:
I think we're better off using standard bootstrap classes `fs-4` or `fs-3` for the font size and `align-middle` on the span rather than custom css.

https://getbootstrap.com/docs/5.0/utilities/text/#font-size
https://getbootstrap.com/docs/5.0/utilities/vertical-align/

Reply:
I updated using fs-2 and align-top as it was a closer match the other home buttons using css. Resolved in commit [7703089](https://github.com/bcgov/parks-reso-public/pull/319/commits/77030898f46724582e766f32e64a9872278558f6)